source/lifecycle/version-archive.rst

```diff
@@ -0,0 +1,742 @@
+Server Version Archive
```

Comment:
@nab-77 & @amyblais - Do version archives for desktop app and mobile apps exist? Do we want to start publishing them now/in a future iteration?

Reply:
Not currently and I'm not sure if we need it. I'll defer to Neil.